src/sinks/kafka.rs

```diff
@@ -97,6 +101,10 @@ impl KafkaSinkConfig {
         if let Some(tls) = &self.tls {
             tls.apply(&mut client_config)?;
         }
+        client_config.set(
+            "compression.codec",
+            &to_string(self.compression.unwrap_or(KafkaCompression::default())),
```

Comment:
I think we could use `unwrap_or_default()` here.

Reply:
Of course.